### ISSUE TITLE
Graph no permission

### ISSUE THREAD

**Issue by timtsj** [catonrug/zabbix-scripts#1]
Hi, can  u help me pls, i have error " no permission" on the reports, i do "chown -R www-data:" and " chmod -R 777" but its dont help me. thx

**Comment by aigarskadikis** [comment]
What script have you used and what operating system lies under the hood?

**Comment by timtsj** [comment]
Zabbix 3.4 PDF report, centos 7, Zabbix 3.4

**Comment by aigarskadikis** [comment]
Hi,
Please try to follow this instruction:
https://catonrug.blogspot.com/2018/04/install-zabbix-dynamic-pdf-report-centos.html

**Comment by timtsj** [comment]
Yes, I followed the instructions, but now the graphs are not displayed
